app/views/works/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<h2>Edit this Work</h2>
+
+
+<%= form_with model: @work do |f| %>
```

Comment:
You have the same form code here and in `new.html.erb`. You should be using a partial to DRY this up.